util/wordpress.js

```diff
@@ -247,7 +247,7 @@ function installTheme(generator, config, done) {
 
 };
 
-function setupTheme(generator, config, done) {
+function setupTheme(generator, config, cb) {
```

Comment:
While I am totally fine with this change, if we want this then we should probably make it to the entire code base, just for consistency.